src/SprykerEco/Zed/Adyen/Business/Hook/Saver/MakePayment/CreditCardSaver.php

```diff
@@ -49,13 +49,14 @@ protected function updatePaymentAdyenTransfer(
      */
     protected function getPaymentStatus(PaymentAdyenTransfer $paymentAdyenTransfer = null): string
     {
-        if (
-            $paymentAdyenTransfer
-            && $paymentAdyenTransfer->getResultCode() === $this->config->getOmsStatusRefused()
-        ) {
-                return $this->config->getOmsStatusRefused();
+        if ($paymentAdyenTransfer->getResultCode() === $this->config->getOmsStatusAuthorized()) {
```

Comment:
This can cause an exception as `$paymentAdyenTransfer` can be `null`.

Reply:
Fixed